### ISSUE TITLE
Competitive Analysis

### ISSUE THREAD

**Issue by gingerbeardman** [glushchenko/fsnotes#217]
I think it's worth looking at the competition to see how they do things.

[Quiver](http://happenapps.com) (free trial, $9.99 full)
_The Programmer's Notebook_
- Pros
  - very polished
  - very user friendly
  - extremely capable
  - mixed content types in the same note!
  - has iPhone companion app (free)
- Cons
  - stores notes in a proprietary format (package of JSON and media)
  - closed source
  - lots of outstanding [issues](https://github.com/HappenApps/Quiver/issues)
  - [slow release schedule](https://twitter.com/happenapps/status/976903361906774022)
  -  iPhone companion app is read only
  -  iPhone companion app does not support iCloud

**Thoughts**
Closed source, semi-abandoned, iOS app not very useful

**Comment by gingerbeardman** [comment]
- [QOwnNotes](http://www.qownnotes.org)
  - a bit overwhelming with the amount of options 
  - but very customisable
  - non-native UI
  - cross platform
  - no iOS companion

**Comment by gingerbeardman** [comment]
[Ulysses](https://ulyssesapp.com) ([subscription](https://ulyssesapp.com/pricing/))
_The Ultimate Writing App for Mac, iPad and iPhone_
- Pros
  - very customisable
  - [fully featured](https://ulyssesapp.com/features/table)
  - cross platform
  - Universal iPhone and iPad companion app
  - award winner
  - steady development
- Cons
  - iCloud only
  - subscription pricing
  - closed source

**Notes**
Good, apart from subscription pricing.

**Comment by gingerbeardman** [comment]
[Agenda](https://agenda.com) (free, paid features)
Agenda is a date-focused note taking app for both planning and documenting your projects.

**Comment by gingerbeardman** [comment]
[Mweb](http://www.mweb.im)
Pro Markdown writing, note taking and static blog generator App

**Comment by jeff-h** [comment]
Before FSNotes, I looked far and wide and couldn't find anything that supported my three essential features:

* individual files stored in the filesystem in a non-proprietary format
* support for RTF instead of just markdown (and ideally a limited RTF; I want bold, strikethrough etc but not multiple fonts or font sizes — as per nvALT and FSNotes)
* super fast search

Also I really dislike monthly/yearly subscription pricing.

In other words I wanted nvALT... except that I want it to actually run properly in 2018 😄 So as far as I have seen, FSNotes is absolutely unique and I am stoked it exists. Still, I agree, worth seeing what niceties could be "borrowed" from other apps.

**Comment by gingerbeardman** [comment]
Came across this one tonight...

http://www.horstmuc.de/wmem.htm

![mempade](https://user-images.githubusercontent.com/49612/42398309-4b48f81a-8161-11e8-8619-c38f75482994.png)

**Comment by jeff-h** [comment]
Sorry, I'm struggling to see anything in that screenshot... 100% of my attention is on that fabulous Windows UI 🤣

**Comment by gingerbeardman** [comment]
Keep It
http://reinventedsoftware.com/keepit/

Apparently this has been in development under a few different names for 14 years (Keep It Together, 2004–; Together, 2007–; Keep It, 2017–).

![shot1](https://user-images.githubusercontent.com/49612/43907287-06338b5e-9bed-11e8-83f9-799dc05e661d.png)

**Comment by bonny** [comment]
These are some very nice similar editors that I've tried:

- [Caret](https://caret.io/) - I like it and use it a lot.
- [iA Writer](https://itunes.apple.com/se/app/ia-writer/id775737590) - Also like and use a lot.
- [Bear](http://www.bear-writer.com/) (Very nice, but does not store notes as markdown and can't sync with Dropbox folder)
- [Typora](https://typora.io/)
- [Letterspace](https://programmerbird.com/letterspace/)
- [Falcon](https://itunes.apple.com/se/app/falcon/id1086447938)

**Comment by jeff-h** [comment]
I think FSNotes is, in many areas, ahead of all those functionally (I read each one's promo spiel but didn't download and try them all — and of course most have their own specific strengths).

However, IMO many of them do have a very nice, polished UI, which really I think they need, to set themselves apart. I think FSNotes' UI will continue to improve but primarily its features set it apart, for me.

**Comment by peavine** [comment]
I had never heard of Falcon, but it's a nice looking app. However, it was last updated in March 2017, which is not encouraging. I do like the app's use of the term Notebook, for what FSNotes calls a storage area.

**Comment by gingerbeardman** [comment]
https://zettelkasten.de/the-archive/

support forum: http://forum.zettelkasten.de

I really like the use of a long reverse date string ("zettel ID", eg. 201802121037) to order/timestamp notes and make cross-reference easier.

**Comment by gingerbeardman** [comment]
Drafts 5
https://getdrafts.com/gettingstarted/overview

![](https://getdrafts.com/images/drafts-devices.png)

**Comment by gingerbeardman** [comment]
FoldingText

https://www.foldingtext.com

Simple
- Plain text
- Outline formatting
- Fold and Focus to see

Surprisingly Adept
- Live formatting
- Scripts, themes, and plugins
- Plain text productivity platform

**Comment by sven-gh** [comment]
Here's another one: https://standardnotes.org
It's open source and often the default suggestion you'll get in privacy-conscious groups

Can't remember why I stopped using it but there must have been something basic that caused friction with me because even though it looks polished I deleted it after 5 minutes

I tried many of the apps suggested above, and loads more I can't all remember.
In the end I gave up and settled for a combination of:
- the default Apple notes app because their sync never causes conflicts / lost content 
- txt files on dropbox which I access with sublime text - I'm no programmer but use sublime because it's fast and does not hiccup plus I like how CMD+C copies the entire line at once... it's the small things that make a big difference

So I'm really excited about FSNotes and hope this will finally allow me to combine things in a single place without getting locked in

**Comment by gingerbeardman** [comment]
The successor to nvALT has been announced.

Codename: nvUltra

https://brettterpstra.com/2019/04/10/codename-nvultra/

> This app works a lot like nvALT (and Notational Velocity, naturally). You pop it up and start typing. Search or create a note in seconds. It has blazing fast and accurate full-text search, the ability to find related notes based on content, and very complete Markdown editing tools (complete with syntax highlighting and theme editing). The biggest difference is that it works with multiple folders and sub-folders. You pick a folder, it indexes it, and you can use it just like nvALT. But then you can open another folder, or create a new one and start editing. It allows you to create folders anywhere, maybe one on Dropbox or iCloud Drive that’s shared, one on an encrypted disk that’s private, one for work, one for home, one for every writing project. You’re not limited to tags (though you can search by and sync with macOS tags within the app), and you can sort your notes into subfolders as well.

**Comment by jeff-h** [comment]
nvUltra: unlike nvALT, it has no RTF and no intention to support it. Just another markdown editor from my point of view.

**Comment by gingerbeardman** [comment]
Inkdrop

- macOS, Windows, Linux
- iOS, Android

https://www.inkdrop.app

**Comment by gingerbeardman** [comment]
StackEdit (free)
- browser-based
- side-by-side preview
- synced scrolling
https://stackedit.io/app

**Comment by gingerbeardman** [comment]
BBEdit 14 (30-day trial; paid $50)

We know that many of our customers create a lot of untitled documents for quick note-taking, and rely on BBEdit's legendary stability and robust crash recovery to protect their work. We've added a new "Notes" feature in BBEdit 14, which provides a large variety of ways to create notes that automatically save themselves, and, perhaps more important, automatically title themselves so that you don't end up wondering which of your 305 "untitled text" documents is the one you're looking for.

https://www.barebones.com/support/bbedit/notes-14.0.html

<img width="432" alt="image" src="https://user-images.githubusercontent.com/49612/128726864-fb598753-d618-4817-b477-a5ec02de8bc9.png">

> If you have been using BBEdit in Free Mode, simply download and install the new version. You’ll get a fresh 30-day evaluation period so that you can try out all the new features, and then BBEdit will return to Free Mode.

**Comment by nth-chile** [comment]
[Obsidian](https://obsidian.md/) is interesting

These aren't necessarily pros, just a list of features:

- uses zettelkasten / "second brain" ideas like backlinks, graphs
- works with just the content of plain text files in the way that FSNotes does
- Electron
- Themable and extensible

**Comment by avichou** [comment]
Upnote (25$ lifetime)

Native kotlin on Android 
Markdown , tags, share link to web
Mac, Android, windows, iOS, ipadOS (and linux too tho not sure )
